helm/ingress-controller/templates/rbac/reserveddomain_viewer_role.yaml

```diff
@@ -0,0 +1,27 @@
+# permissions for end users to view reserveddomains.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: clusterrole
+    app.kubernetes.io/instance: reserveddomain-viewer-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: ngrok-ingress-controller
+    app.kubernetes.io/part-of: ngrok-ingress-controller
+    app.kubernetes.io/managed-by: kustomize
```

Comment:
:(

Reply:
Good catch on the auto-generated stuff. I'll fix those up